src/pointer-analysis/value_set_dereference.cpp

```diff
@@ -224,6 +261,15 @@ exprt value_set_dereferencet::dereference(const exprt &pointer)
   if(compare_against_pointer != pointer)
     value = let_exprt(to_symbol_expr(compare_against_pointer), pointer, value);
 
+  if(display_points_to_sets)
+  {
+    std::stringstream ss;
+    ss << format(value);
+    json_result["Value"] = json_stringt(ss.str());
+
+    std::cout << ",\n" << json_result;
```

Comment:
🚫 This is not allowed. Pass a message handler to the class instead as done everywhere ese.

Reply:
Fixed this.